modules/templates/util_avatar.go

```diff
@@ -57,6 +57,8 @@ func (au *AvatarUtils) Avatar(item any, others ...any) template.HTML {
 		if src != "" {
 			return AvatarHTML(src, size, class, t.AsUser().DisplayName())
 		}
+	default:
+		return AvatarHTML(avatars.DefaultAvatarLink(), size, class, "")
```

Comment:
It seems that the code could be:

```
switch () {
...
}
return return AvatarHTML(avatars.DefaultAvatarLink(), size, class, "")
```

The old code (returning an empty string) doesn't seem useful, instead if might break some layouts which expects to see an `img`.

Reply:
Done.